### DISCUSSION TITLE
typos in "reloadpackage" command

### DISCUSSION THREAD

**Discussion by aoyamajia** [FakeFishGames/Barotrauma#13882] [Bug reports]
### What happened?

The warning for the `reloadpackage` command talks about `reloadcorepackage`. Some users may read too well and end up running that command instead.
Also typo "...may break **thingg**, to do it anyway..."
![image](https://github.com/FakeFishGames/Barotrauma/assets/39614559/e6674b9e-92c3-42e8-854e-3ff803787bae)

:OlafDog:

### Reproduction steps

Attempt to reload a mod in editor without force and notice the misleading warning

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/2d9054aba5985dd0b1c4f459c2af0c587f5c3d39 (commit in our private repo)
